mmv1/templates/terraform/examples/firebasehosting_site_basic.tf.erb

```diff
@@ -0,0 +1,6 @@
+resource "google_firebase_hosting_site" "default" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  site = "tf-test-site%{random_suffix}"
+  app_id = "<%= ctx[:vars]['appId'] %>"
```

Comment:
we probably want a 2nd resource without an app_id to show it clearly is optional (that should probably be the default one as it is the simplest).

Reply:
Good idea, done